bundles/org.openhab.core.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/item/EnrichedItemDTOMapper.java

```diff
@@ -93,41 +93,50 @@ private static EnrichedItemDTO map(Item item, ItemDTO itemDTO, URI uri, boolean
         return enrichedItemDTO;
     }
 
-    private static StateDescription considerTransformation(StateDescription stateDescription) {
-        if (stateDescription != null) {
-            String pattern = stateDescription.getPattern();
-            if (pattern != null) {
-                try {
-                    return TransformationHelper.isTransform(pattern)
-                            ? StateDescriptionFragmentBuilder.create(stateDescription).withPattern(pattern).build()
-                                    .toStateDescription()
-                            : stateDescription;
-                } catch (NoClassDefFoundError ex) {
-                    // TransformationHelper is optional dependency, so ignore if class not found
-                    // return state description as it is without transformation
-                }
+    private static StateDescription considerTransformation(@Nullable StateDescription desc) {
+        if (desc == null) {
+            return null;
+        }
+        String pattern = desc.getPattern();
+        if (pattern == null) {
+            return desc;
+        } else {
+            try {
+                return TransformationHelper.isTransform(pattern)
+                        ? new StateDescription(desc.getMinimum(), desc.getMaximum(), desc.getStep(), "",
```

Comment:
I need to cheery pick #655 changes and apply them to this file again.

Reply:
Wouldn't it make sense to rebase the working branch on a regular basis?

Reply:
I rebased, but for this particular file I chose to just take mine to not break the PR. But will reapply the made changes.

Reply:
Why will this PR be broken if you rebase to the master?

Reply:
Again: I have rebased. Just for this one file I had conflicts that I didn't feel comfortable to hackish resolve them, so I just kept my changes, but annotated with a comment to remind me to cheery pick the changes back.